dist/t/spec/support/capybara.rb

```diff
@@ -24,7 +24,7 @@
 
 # Set hostname
 begin
-  hostname = Socket.gethostbyname(Socket.gethostname).first
+  hostname = Addrinfo.getaddrinfo(Socket.gethostname).first
 rescue SocketError
```

Comment:
Does still make sense to rescue from a SocketError?

Reply:
https://ruby-doc.org/stdlib-3.0.0/libdoc/socket/rdoc/Addrinfo.html#method-c-getaddrinfo does not list it, but a quick test showed that it still throws a
```
SocketError (getaddrinfo: No address associated with hostname)
```

Reply:
> https://ruby-doc.org/stdlib-3.0.0/libdoc/socket/rdoc/Addrinfo.html#method-c-getaddrinfo does not list it, but a quick test showed that it still throws a
> 
> ```
> SocketError (getaddrinfo: No address associated with hostname)
> ```

Sorry for not understanding it. What do you mean by "a quick test"?

Reply:
If you go into an IRB session and try to resolve a hostname that does not exist, you'll see that it throws a `SocketError` error